src/messages/TextMessage.vue

```diff
@@ -1,37 +1,47 @@
 <template>
-  <div class="sc-message--text" :style="messageColors">
+  <div class="sc-message--text" v-bind:class="{ 'confirm-delete' : deleteConfirmation == true }" :style="messageColors">
```

Comment:
Strict equality (`===`) should be used normally, but since this is a comparison with a boolean, it's not needed at all: `{ 'confirm-delete' : deleteConfirmation }` will suffice. Renaming the variable from `deleteConfirmation` to `confirmDeletion` might also help.

Following the existing styles, `:class` should be used instead of `v-bind:class`. No spaces inside `{}`.